### PR TITLE
Swift rules update

### PR DESCRIPTION
**Pull request by zippy1978**
Here is a PR cleaning up Swift rules (duplicates, missing).
The goal is to be in sync with this PR: https://github.com/green-code-initiative/ecoCode-ios/pull/22

@dedece35 @jycr

### CODE REVIEW

File: ecocode-rules-specifications/README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+git
```
Comment (dedece35) [inline ecocode-rules-specifications/README.md:1]: I don't understand this added line in this file ... why ?

**Review by dedece35 — CHANGES_REQUESTED** [review]

File: ecocode-rules-specifications/README.md (hunk 1 above)
Comment (zippy1978) [reply ecocode-rules-specifications/README.md]: Oh sorry, it's a mistake.
I removed it

**Review by dedece35 — APPROVED** [review]

**Review by dedece35 — CHANGES_REQUESTED** [review]
please update CHANGELOG.md to add a quick description of your modifications

**Comment by zippy1978** [comment]
@dedece35 CHANGELOG.md updated !

File: ecocode-rules-specifications/src/main/rules/EC14/EC14.json [hunk 2]
```diff
@@ -1,20 +0,0 @@
-{
```
Comment (dedece35) [inline ecocode-rules-specifications/src/main/rules/EC14/EC14.json:1]: @zippy1978 ... la suppression de tous les fichiers de cet PR est bien voulue ? car doublon avec d'autres rules, c'est ça ?

Comment (zippy1978) [reply ecocode-rules-specifications/src/main/rules/EC14/EC14.json:1]: Yes!
The rules deleted where originally added by @gaelfoppolo at the very beginning of the repo. But after that they were re-added with other ids.
So the goal is t clean up the old ones (used by no plugin at all!), and keep the new ones (used by android plugin, and now iOS)

**Review by dedece35 — APPROVED** [review]

**Pull request merged**